includes/admin/admin-commands.php

```diff
@@ -0,0 +1,87 @@
+<?php
+/**
+ * SCF Commands Integration
+ *
+ * @package Secure Custom Fields
+ */
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly.
+}
+
+/**
+ * Initializes SCF commands integration
+ *
+ * This function handles the integration with WordPress Commands (Cmd+K / Ctrl+K),
+ * providing navigation commands for SCF admin pages and custom post types.
+ *
+ * The implementation follows these principles:
+ * 1. Only loads in screens where WordPress commands are available.
+ * 2. Performs capability checks to ensure users only see commands they can access.
+ * 3. Core administrative commands are only shown to users with SCF admin capabilities.
+ * 4. Custom post type commands are conditionally shown based on edit_posts capability
+ *    for each specific post type.
+ * 5. Post types must have UI enabled (show_ui setting) to appear in commands.
+ *
+ * @since 6.5.0
+ */
+function acf_commands_init() {
+	// Only load on admin screens.
+	if ( ! is_admin() ) {
+		return;
+	}
+
+	// Ensure we only load our commands where the WordPress commands API is available.
+	if ( ! wp_script_is( 'wp-commands', 'registered' ) ) {
+		return;
+	}
+
+	$custom_post_types = array();
+
+	if ( function_exists( 'acf_get_acf_post_types' ) ) {
```

Comment:
Why this check, can't we assume it exists since we're in the SCF codebase?

Reply:
There's an option to disable post type entirely, but checking the codebase confirms the function is loaded unconditionally (just returning empty if post types are disabled), so it is indeed safe to assume it always exists. 👍